cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,23 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io/#/login');
+  });
+
   it('should provide an ability to log in', () => {
```

Comment:
I don't think we needed to check for the "Sign In" text on the page.
There is no such condition in the task